src/logic/hero-logic.ts

```diff
@@ -541,9 +541,9 @@ Complex or time-consuming tests might require an action if made in combat—or c
 	static getSpeed = (hero: Hero) => {
 		let value = 5;
 
-		const features = this.getFeatures(hero).filter(f => f.type === FeatureType.Size);
+		const features = this.getFeatures(hero).filter(f => f.type === FeatureType.Speed);
 		if (features.length > 0) {
-			const datas = features.map(f => f.data as FeatureSpeedData);
+			const datas = features.map(f => f.data);
 			value = Collections.max(datas.map(d => d.speed), v => v) || 0;
 		}
```

Comment:
using the strong data type found this bug with trying to get speed from the `Size` feature